.github/ISSUE_TEMPLATE/2_bug-or-performance-issue-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug or performance issue report
+about: Create a report to help us improve oneDPL
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Please add a colon after your bolded headers and be consistent on capitalization: **Describe the Bug:**

Reply:
Done.